pkg/core/blockchain.go

```diff
@@ -514,6 +514,11 @@ func (bc *Blockchain) storeBlock(block *block.Block) error {
 			v := bc.spawnVMWithInterops(systemInterop)
 			v.SetCheckedHash(tx.VerificationHash().BytesBE())
 			v.LoadScript(t.Script)
+			v.SetPriceGetter(getPrice)
+
+			gasAmount := gasForFree + t.Gas
+			v.SetGasLimit(int(gasAmount / util.Fixed8(interopGasRatio)))
```

Comment:
We also need some inifinite-gas mode, we're gonna be using it for neofs, for example to avoid messing with gas.

Reply:
I have added `FreeGasLimit` which can be set to zero if needed.